tests/unit/test_service.py

```diff
@@ -228,6 +229,38 @@ def _fake_measurement_function():
     pass
 
 
+@pytest.mark.parametrize(
+    "service_config,provided_interfaces",
+    [
+        (
+            "example.serviceconfig",
+            [
+                "ni.measurementlink.measurement.v1.MeasurementService",
+                "ni.measurementlink.measurement.v2.MeasurementService",
+            ],
+        ),
+        ("examples.v1.serviceconfig", ["ni.measurementlink.measurement.v1.MeasurementService"]),
+        ("examples.v2.serviceconfig", ["ni.measurementlink.measurement.v2.MeasurementService"]),
+    ],
+)
+def test__measurement_service__create_measurement_service__service_info_populated_by_serviceconfig(
+    service_config: str, provided_interfaces: List[str]
+):
+    assets_directory = pathlib.Path(__file__).resolve().parent.parent / "assets"
```

Comment:
Please copy this fixture to `tests/conftest.py` and use it: https://github.com/ni/nidaqmx-python/blob/master/tests/conftest.py#L250